src/Directory.Build.targets

```diff
@@ -54,4 +54,43 @@
   <Target Name="DumpTargets" BeforeTargets="ResolveProjectReferences">
     <Message Text="DumpTargets> $(OutputPath), C=[$(Configuration)], CG=[$(ConfigurationGroup)], OG=[$(OSGroup)], TG=[$(TargetGroup)]" Importance="Low" />
   </Target>
+
+  <PropertyGroup Condition="'$(IsInvokableTestProject)' == 'true'">
+    <LaunchSettingsOutputFileFullPath Condition="'$(LaunchSettingsOutputFileFullPath)' == ''">$(AppDesignerFolder)/launchSettings.json</LaunchSettingsOutputFileFullPath>
+    <LaunchSettingsInputFileFullPath Condition="'$(LaunchSettingsInputFileFullPath)' == ''">$(ToolsDir)tests.targets</LaunchSettingsInputFileFullPath>
+
+    <CompileDependsOn>
+      GenerateLaunchSettingsFile;
+      $(CompileDependsOn);
+    </CompileDependsOn>
+  </PropertyGroup>
+
+  <!-- Create debug support files -->
+  <Target Name="GenerateLaunchSettingsFile"
+          Condition="'$(LaunchSettingsOutputFileFullPath)' != '' AND '$(OmitLaunchSettings)' != 'true'"
+          Inputs="$(LaunchSettingsInputFileFullPath)"
+          Outputs="$(LaunchSettingsOutputFileFullPath)">
+
+    <PropertyGroup>
+      <LaunchSettingsFileContent>
+{
+  &quot;profiles&quot;: {
+    &quot;xUnit.net Console&quot;: {
+      &quot;commandName&quot;: &quot;Executable&quot;,
+      &quot;executablePath&quot;: &quot;$(StartProgram.Replace('\', '\\'))&quot;,
+      &quot;commandLineArgs&quot;: &quot;$(StartArguments)&quot;,
+      &quot;workingDirectory&quot;: &quot;$(StartWorkingDirectory.Replace('\', '\\'))&quot;
+    }
+  }
+}
+      </LaunchSettingsFileContent>
+    </PropertyGroup>
+
+    <MakeDir Directories="$(AppDesignerFolder)" />
+    <WriteLinesToFile File="$(LaunchSettingsOutputFileFullPath)"
```

Comment:
Do we want to add the file to `@(FileWrites)` so it gets cleaned during a `clean`?

Reply:
> There are two drawbacks to doing this, however. Files to be deleted must reside under the output path, and you must append items to that list before the clean cache is written to disk, which occurs in either the Clean or IncrementalClean target, depending on the type of build that is executed. These targets will be called before the Build target completes.

https://msdn.microsoft.com/en-us/magazine/dd419659.aspx

FileWrites doesn't work, I will try another way.

Reply:
Ok, then I wouldn't worry about it. I figured if it was simple to add it. But since it isn't, let's just forget it.

Reply:
I added a clean target and added it to `CleanDependsOn` in the commit that I just pushed.